spec/requests/image_file_sizes_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+describe "Image File Sizes", type: :request do
+  include Webpacker::Helper
```

Comment:
Is this include required here ? Not sure if any webpacker helpers are used in this specific test.